backend/LeetWars.Notifier/LeetWars.Notifier.WebAPI/Services/ConsumerServiceFactory.cs

```diff
@@ -0,0 +1,31 @@
+﻿using LeetWars.Notifier.WebAPI.Interfaces;
+using LeetWars.Notifier.WebAPI.Models.Helpers;
+using LeetWars.RabbitMQ;
+using LeetWars.RabbitMQ.Services;
+
+namespace LeetWars.Notifier.WebAPI.Services;
+
+public class ConsumerServiceFactory : IConsumerServiceFactory
```

Comment:
why do we need this factory?

Reply:
Because we have different consumer services that implement the same interface, and this factory return that one instance that our service needs

Reply:
I believe you misunderstand what "consumer" mean